chart/values.yaml

```diff
@@ -130,7 +136,26 @@ kube-prometheus-stack:
   alertmanager:
     alertmanagerSpec:
       replicas: 3
-
+      ## AlertManager resource requests
+      resources:
+        limits:
+          memory: 100Mi
+          cpu: 100m
+        requests:
+          memory: 50Mi
+          cpu: 4m
```

Comment:
Just confirming you actually meant 4m? not 40m?

Reply:
Yes, `4m` is correct here. This value is a result of tests done by OpenShift team.